packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -149,9 +151,52 @@ export function setValueForProperty(
   if (shouldIgnoreAttribute(name, propertyInfo, isCustomComponentTag)) {
     return;
   }
+
+  if (
+    enableCustomElementPropertySupport &&
+    isCustomComponentTag &&
```

Comment:
Does this need to be restricted to just custom components? It's possible to trigger custom events on any element and there are many reasons to do so.

Reply:
I know that's what preact does, but I'm worried about breaking existing usage with regular elements. There was already significant concern with how this reserves everything starting with "on" for custom events, like anything starting with "one" and "once" for example.